### PR TITLE
Update Stats Retrieval 

### PR DESCRIPTION
**Pull request by nrwslac**
- lcls-twincat-optics running 0.4.1
- changed stats for all axes currently using `FB_RMSWatch` to use raw encoder counts
- added `FB_HomsStats` to both gantries on each mirror mr1l0 and mr2l0

### CODE REVIEW

**Comment by klauer** [comment]
Pairs with https://github.com/pcdshub/lcls-twincat-optics/pull/14

File: lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj [hunk 1]
```diff
@@ -90,6 +90,10 @@
     <Folder Include="POUs\Serial Comm" />
   </ItemGroup>
   <ItemGroup>
+    <PlaceholderReference Include="LCLS General">
+      <DefaultResolution>LCLS General, * (SLAC)</DefaultResolution>
```
Comment (ZLLentz) [inline lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: What's going on here? We shouldn't have a */latest general reference, it should be a specific tag

Comment (nrwslac) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: Set LCLS General -> 2.6.0

Comment (ZLLentz) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: Ok, great! I still see something weird in the diff here where this comment change is attached but I won't pretend to understand everything about how these files are structured.

**Review by ZLLentz — APPROVED** [review]
LGTM like the other one

File: lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj (hunk 1 above)
Comment (nrwslac) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: I noticed this as well. Not sure why its still here.

Comment (ZLLentz) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: If it causes a weird build issue maybe we'll learn something new. If things look normal in the UI let's move on with our lives.

Comment (nrwslac) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: Screen Shot 2022-08-25 at 3 04 43 PM

Comment (nrwslac) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: I see PMPS, 2.0.0 using LCLS General, * would that be an issue?

Comment (ZLLentz) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: It should be OK, the libraries are intended to use * which means "match the static versions used elsewhere". The effective version matches so it should be fine.

Comment (nrwslac) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: I did test the IOC build which is working now, but at some point I did have to delete the LCLS General reference and re-add to alleviate a pytmc lint error which complained about some duplications.

Comment (nrwslac) [reply lcls-plc-lfe-optics/lfe_optics/lfe_optics.plcproj:94]: I'll call this good, merge and set the IOC to a release.

**Pull request merged**